app/experimenter/experiments/tests/test_models.py

```diff
@@ -709,6 +719,30 @@ def test_completed_all_sections_true_when_complete(self):
         )
         self.assertTrue(experiment.completed_all_sections)
 
+    def test_completed_all_sections_true_for_pref_study_without_addon(self):
+        experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_REVIEW,
+            type=Experiment.TYPE_PREF,
+            addon_release_url=None,
+        )
+        self.assertTrue(experiment.completed_all_sections)
+
+    def test_completed_all_sections_false_for_addon_study_without_addon(self):
```

Comment:
This test duplicates the test in `test_addons_is_complete_when_release_url_set` above. I think keeping this one is OK.

Reply:
What do you mean?  One is testing the `completed_addon` property, the other is testing the `completed_all_sections` property's optional dependence on the addon fields for addon studies.

Reply:
I mean that you don't need to have a dedicated unit test just for the property method. If the `completed_all_sections` test works, it'll travel all code branches. 

But if you want to keep both, that's fine too. Not a blocker for me.

Reply:
It's specifically testing the case where it's an addon type study and the completed_addon field becomes required here: https://github.com/mozilla/experimenter/pull/1064/files#diff-562e6046de58c57f7e5e7f6ecdb15efbR473